### PR TITLE
Add two more test spans to try isolate long add fallible span

### PR DESCRIPTION
**Pull request by Restioson**
I'm very stuck with this issue since it I can't reproduce locally at all, so I thought I'd add some more debug spans so we can see what it's doing better.

### CODE REVIEW

File: maker/src/connection.rs [hunk 1]
```diff
@@ -478,7 +480,7 @@ impl Actor {
                                         "we are not connected to ourselves, this should really not happen",
                                     )?;
                             }
-                        }
+                        }.instrument(tracing::debug_span!(parent: Span::none(), "Test long span"))
```
Comment (klochowicz) [inline maker/src/connection.rs]:
```suggestion
                        }.instrument(tracing::debug_span!(parent: Span::none(), "Fallible task handler span"))
```

File: maker/src/connection.rs [hunk 2]
```diff
@@ -490,7 +492,7 @@ impl Actor {
                                     error,
                                 })
                                 .await;
-                        }
+                        }.instrument(tracing::debug_span!(parent: Span::none(), "Test error span"))
```
Comment (klochowicz) [inline maker/src/connection.rs]:
```suggestion
                        }.instrument(tracing::debug_span!(parent: Span::none(), "Fallible task error span"))
```

**Review by klochowicz — APPROVED** [review]
thanks! I've only suggested more permanent span names that should be unambiguous.

**Comment by Restioson** [comment]
I've incorporated the suggestions and rebased this on top of master

**Pull request merged**